src/framework/components/particlesystem/particlesystem_system.js

```diff
@@ -195,12 +195,49 @@ pc.extend(pc, function() {
                 type: 'boolean',
                 defaultValue: false
             }, {
-                name: "spawnBounds",
-                displayName: "Spawn Bounds",
-                description: "Defines an AABB in which particles are allowed to spawn",
+                name: "emitterShape",
+                displayName: "Emitter Shape",
+                description: "Defines a shape in which particles are allowed to spawn",
+                type: "enumeration",
+                options: {
+                    enumerations: [{
```

Comment:
Should we add a pc.EMITTERSHAPE_NONE option which would be the default? That way we can hide the emitterExtents completely. That way we can set the default emitterExtents to 1, 1, 1 which is more meaningful than 0, 0, 0 (which is like that because we don't have EMITTERSHAPE_NONE)